scss/4-elements/_tables.scss

```diff
@@ -0,0 +1,129 @@
+.cads-table-container {
+    width: 100%;
+    overflow: auto;
+}
+
+table,
+.cads-table {
+    text-align: left;
+    border-spacing: 0;
+    width: 100%;
+    margin-bottom: $cads-spacing-4;
+
+    @include cads-border(
+        a,
+        $cads-border-width-small,
+        $cads-border-radius-large,
+        $cads-language__input-border-colour
+    );
+
+    border-bottom: none;
+
+    caption,
+    .cads-table-caption {
+        @extend .cads-heading-small;
+
+        margin-bottom: $cads-spacing-2;
+        text-align: left;
+    }
+
+    th,
+    td {
+        padding: $cads-spacing-2;
+        font-weight: $cads-font-weight__regular;
+
+        @include cads-border(
+            b,
```

Comment:
what's b and a here? Can you use descriptive name

Reply:
a/t/p/h/l/r guess what it does

We could but then we'll need to go change everything. TBH I'm not sure this cads-border mixin is necessary overall.

Reply:
Mixin removed as discussed on Slack.